addons/mail/models/discuss/discuss_channel_rtc_session.py

```diff
@@ -21,7 +21,7 @@ class DiscussChannelRtcSession(models.Model):
 
     channel_member_id = fields.Many2one('discuss.channel.member', required=True, ondelete='cascade')
     channel_id = fields.Many2one('discuss.channel', related='channel_member_id.channel_id', store=True, readonly=True)
-    partner_id = fields.Many2one('res.partner', related='channel_member_id.partner_id', string="Partner")
+    partner_id = fields.Many2one('res.partner', related='channel_member_id.partner_id', string="Partner", store=True, index=True)
```

Comment:
I don't know if we need an upgrade script to store the values in this case or if it can do it automatically.

Reply:
I think you need upgrade script

Reply:
I remember a case recently that looked like this one where I'm sure it was necessary before but I was surprised that it works automatically now. Just a note to double check which it is now.

Reply:
![image](https://github.com/user-attachments/assets/517f42f2-0400-4d70-8205-9cc8a4364be6)
Apparently new store field are not listed anymore, and I wouldn't say this field requires complex computation.